README.md

```diff
@@ -1,141 +1,54 @@
-As such, all module interactions are abstracted through the [ModuleTemplate](api/v1beta1/moduletemplate_types.go).
-
-This abstraction of a template is used for generically deploying instances of a module within a Kyma Runtime at a specific Release Group we call `Channel` (for more information, visit the respective Chapter in the [Concept for Modularization](https://github.com/kyma-project/community/tree/main/concepts/modularization#release-channels)). It contains not only a specification of a Module with it's different components through [OCM Component Descriptors](https://github.com/gardener/component-spec/blob/master/doc/proposal/02-component-descriptor.md).
-
-These serve as small-scale BOM's for all contents included in a module and can be interpreted by Lifecycle Manager and [Module Manager](https://github.com/kyma-project/module-manager/)
-to correctly install a module. (for more information, please have a look at the respective chapter in the [Kyma Modularization Concept](https://github.com/kyma-project/community/tree/main/concepts/modularization#component-descriptor))
-
-### Versioning and Releasing
-
-Kyma up to Version 2.x was always a single release. However, the vision of lifecycle-manager is to fully encapsulate individual Modules, with each providing a (possibly fully independent) Release Cycle.
-However, Control-Plane deliveries are by design continuously shipped and improved. As a result, even if we will continue to support versioned Module Deliveries, the Lifecycle-Manager and its adjacent infrastructure will be maintained and delivered continously and it is recommended to track upstream as close as possible.
-
-### Comparison to the Old Reconciler
-
-Traditionally, Kyma was installed with the [Kyma Reconciler](https://github.com/kyma-incubator/reconciler), a Control-Plane implementation of our architecture based on polling and a SQL Store for tracking reconciliations.
-While this worked great for smaller and medium scale deliveries, we had trouble to scale and maintain it when put under significant load.
-We chose to replace this with Operator-focused Reconciliation due to various reasons, more details on the reasoning can be found in our [Concept for Operator Reconciliation](https://github.com/kyma-project/community/tree/main/concepts/operator-reconciliation)
-
-## Testing and implementation guide for Lifecycle Manager developers
-
-- For our detailed overview and reference documentation refer to our [reference overview](docs/developer/reference-documentation.md)
-- For a detailed cluster and module setup refer to our [test environment guide](docs/developer/local-test-setup.md)
-- For configuring the lifecycle-manager operator refer to our [developer guide](docs/user/starting-operator-with-webhooks.md)
+To find more details about Lifecycle Manager and its components, go to the [`docs`](/docs/) directory.
```

Comment:
I would make this point stand out, we already have the documentation index there so it would make it easier for the user to continue the docu journey